### ISSUE TITLE
[BUG] some modules duplicate if you click them a few times

### ISSUE THREAD

**Issue by ENDER1355** [CCBlueX/LiquidBounce#522]
**Describe the bug**
some modules duplicate if you click them a few times
**To Reproduce**
1. click on a module (badwifi)
2. click it again
3. See the error

**Screenshots**

**Additional information:**
 - OS: Windows 10 64 bit 21h1
 - LiquidBounce Build Version NextGen
 - Minecraft Version 1.16.5

**Comment by 1zun4** [comment]
Please be aware that nextgen is still in early development and bugs like this can occur.

**Comment by ENDER1355** [comment]
oh ok, is reporting these bugs good?

**Comment by BobIsMyManager** [comment]
Obviously, I just think that he says that because people keep complaining about bugs, crashes
